### PR TITLE
add a way to skip rest calibration tutorial

### PR DESCRIPTION
**Pull request by ImUrX**
Adds a 10s timer that makes appear the skip button, and also doesnt show calibration tutorial for mocopi or haritora

### CODE REVIEW

**Review by loucass003 — CHANGES_REQUESTED** [review]
Aside for the one comment, this looks good

File: gui/src/hooks/imu-logic.ts [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 ): boolean {
   const imuExists = useMemo(
-    () => connectedTrackers.some((tracker) => tracker.tracker.info?.isImu),
+    () =>
+      connectedTrackers.some(
+        (tracker) =>
+          tracker.tracker.info?.isImu &&
+          !(
+            tracker.device?.hardwareInfo?.boardType &&
+            IGNORED_BOARDS.has(tracker.device?.hardwareInfo?.boardType as string)
```
Comment (ButterscotchV) [inline gui/src/hooks/imu-logic.ts:16]: I would suggest using `McuType` but it doesn't look like we have one for Haritora, so whatever

**Review by ButterscotchV — COMMENTED** [review]
Looks good besides the comment from earlier

File: gui/src/hooks/imu-logic.ts (hunk 1 above)
Comment (ImUrX) [reply gui/src/hooks/imu-logic.ts:16]: but its wrong to call haritora an mcutype, its a DA14

**Review by ButterscotchV — APPROVED** [review]

**Review by loucass003 — APPROVED** [review]

**Pull request merged**